### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by benardop**
# Hello Microverse
In this project I have:
-   Added html/css linters
-   Added index.html file
-   Added styles.css
-  Added .gitignore file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+# Stores VSCode versions used for testing VSCode extensions
+.vscode-test
+
+# yarn v2
+.yarn/cache
+.yarn/unplugged
+.yarn/build-state.yml
+.yarn/install-state.gz
+.pnp.*
```
Comment (zieeco) [inline .gitignore]: - Please, kindly add the `node_modules/`directory and `test.md` file.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi, @benardop, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Great start :+1: 


## Required Changes ♻️

- [ ] Kindly add a description to your PR. You can check the image below to see how to add it. 
ds
- [ ] Please, update the title of your PR to be more descriptive e.g. `Hello Microverse`

- [ ] Please, kindly remove the linter file you've added and use the linter from [microverse html-css](https://github.com/microverseinc/linters-config/tree/master/html-css) linter, add the `.github/workflows`, `.hintrc`, and `.stylelintrc.json` to the root of your project and ensure there is no linter error.

- [ ] Also, kindly add a readme file to your repo. You can use the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)

_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (benardop) [reply .gitignore]: @zieeco I thought node_modules/ and test.md should be added in the .gitignore file? Kindly clarify that.

Comment (benardop) [reply .gitignore]: @zieeco Can you please kindly check once again, I already have README.med file in my GitHub repository which I have created by following the template given by Microverse.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @benardop 

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- Good commit messages  ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] You may please add the readme file to your branch files, not the microverse template files. I can't seem to see the readme on your branch files
picture1


## Optional suggestions

- [ ] As the previous reviewer suggested, please update the Pull request title as well. Not only the description. `Hello Microverse` was an example to replace the add-linters text copied below.
Picture2


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approve :green_circle: :tada:

Your project is complete. There is nothing else to say about your project. :+1: 
Go ahead and merge it :100:

[Optional]
- [ ] Please add an [MIT](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/MIT.md) file to your project because it's your license file for the project :blush:
- [ ]  Please add all sections to the readme. You can see this [readme](https://github.com/HAMMAS-SALEEM/Hello-Microverse/blob/master/README.md) :+1:

**Pull request merged**